Tasks/CondaEnvironmentV0/task.json

```diff
@@ -11,35 +11,37 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
+        "Major": 1,
         "Minor": 136,
         "Patch": 0
     },
     "demands": [],
-    "instanceNameFormat": "Conda Environment $(environmentName)",
-    "groups": [
-        {
-            "name": "advanced",
-            "displayName": "Advanced",
-            "isExpanded": false
-        }
-    ],
+    "instanceNameFormat": "Conda environment $(environmentName)",
     "inputs": [
+        {
+            "name": "customEnvironment",
```

Comment:
Maybe "createCustomEnvironment" so that it sounds more boolean when it appears in YAML.

Reply:
ok